### PR TITLE
Update Switch.js

### PR DESCRIPTION
**Pull request by PaitoAnderson**
Just a little code cleanup

### CODE REVIEW

File: lib/Switch.js [hunk 1]
```diff
@@ -82,8 +82,7 @@ export class Switch extends Component {
 	animateSwitch(value, cb = () => { }) {
 		Animated.parallel([
 			Animated.spring(this.state.transformSwitch, {
-				toValue: value ? this.props.circleSize / 2 : -this.props.circleSize / 2,
-				duration: 200
```
Comment (shahen94) [inline lib/Switch.js]: Why `duration` has been removed ?

File: lib/Switch.js [hunk 2]
```diff
@@ -143,11 +142,11 @@ export class Switch extends Component {
 							{ transform: [{ translateX: transformSwitch }], width: circleSize * 2 },
 						]}
 					>
-						<Text style={[styles.text, styles.inactiveText, styles.paddingRight, activeTextStyle]}>
```
Comment (shahen94) [inline lib/Switch.js]: Can you please explain this change ?

File: lib/Switch.js [hunk 3]
```diff
@@ -143,11 +142,11 @@ export class Switch extends Component {
...
+						<Text style={[styles.text, styles.paddingRight, activeTextStyle]}>
 							{activeText}
 						</Text>
 						<Animated.View style={[styles.circle, { backgroundColor: interpolatedCircleColor, width: circleSize, height: circleSize, borderRadius: circleSize / 2 }]} />
-						<Text style={[styles.text, styles.active, styles.paddingLeft, inactiveTextStyle]}>
```
Comment (shahen94) [inline lib/Switch.js]: Can you please explain this change ?

File: lib/Switch.js (hunk 2 above)
Comment (PaitoAnderson) [reply lib/Switch.js]: Just removing `styles.inactiveText` which isn't defined in the StyleSheet object.

File: lib/Switch.js (hunk 3 above)
Comment (PaitoAnderson) [reply lib/Switch.js]: Just removing `styles.active` which isn't defined in the StyleSheet object.

File: lib/Switch.js (hunk 1 above)
Comment (PaitoAnderson) [reply lib/Switch.js]: As far as I know `duration` isn't property on `Animated.spring`? https://facebook.github.io/react-native/docs/animated.html#spring

**Pull request merged**